### PR TITLE
Fix typo

### PR DESCRIPTION
**Pull request by e-kwsm**

### CODE REVIEW

File: Code/GraphMol/Aromaticity.cpp [hunk 1]
```diff
@@ -158,11 +158,11 @@ static bool applyHuckel(ROMol &mol, const INT_VECT &ring,
                         const VECT_EDON_TYPE &edon);
 
 static void applyHuckelToFused(
-    ROMol &mol,                   // molecule of interets
+    ROMol &mol,                   // molecule of interests
```
Comment (greglandrum) [inline Code/GraphMol/Aromaticity.cpp]:
```suggestion
    ROMol &mol,                   // molecule of interest
```

File: Code/GraphMol/Aromaticity.cpp [hunk 2]
```diff
@@ -333,18 +333,18 @@ bool applyHuckel(ROMol &mol, const INT_VECT &ring, const VECT_EDON_TYPE &edon,
 }
 
 void applyHuckelToFused(
-    ROMol &mol,                   // molecule of interets
+    ROMol &mol,                   // molecule of interests
```
Comment (greglandrum) [inline Code/GraphMol/Aromaticity.cpp]:
```suggestion
    ROMol &mol,                   // molecule of interest
```

File: Code/GraphMol/Depictor/DepictUtils.cpp [hunk 3]
```diff
@@ -216,7 +216,7 @@ RDKit::INT_VECT findNextRingToEmbed(const RDKit::INT_VECT &doneRings,
   // in common are in general flat systems to start with and can be embedded
   // cleanly.
   // when there are more than 2 atoms in common, these are most likely bridged
-  // syste, which are
+  // system, which are
```
Comment (greglandrum) [inline Code/GraphMol/Depictor/DepictUtils.cpp]:
```suggestion
  // systems, which are
```

File: Code/GraphMol/Depictor/EmbeddedFrag.cpp [hunk 4]
```diff
@@ -246,7 +246,7 @@ EmbeddedFrag::EmbeddedFrag(const RDKit::Bond *dblBond) {
   // used to deal with this ugliness.
   // Now we will just embed the atoms on the double bonds and mark at these
   // atoms
-  // the direction in which the in comming single bonds should go.
+  // the direction in which the in coming single bonds should go.
```
Comment (greglandrum) [inline Code/GraphMol/Depictor/EmbeddedFrag.cpp]:
```suggestion
  // the direction in which the incoming single bonds should go.
```

File: Code/GraphMol/Depictor/RDDepictor.cpp [hunk 5]
```diff
@@ -196,7 +196,7 @@ void computeInitialCoords(RDKit::ROMol &mol,
 
   efrags.clear();
 
-  // user specfied coordinates exist
+  // user specified coordinates exist
```
Comment (greglandrum) [inline Code/GraphMol/Depictor/RDDepictor.cpp]:
```suggestion
  // user-specified coordinates exist
```

File: Code/GraphMol/Depictor/RDDepictor.cpp [hunk 6]
```diff
@@ -384,25 +384,25 @@ unsigned int compute2DCoords(RDKit::ROMol &mol,
   other. The second component is the sum of squares of the
   difference in distance between those in dmat and the generated
   structure.  The user can adjust the relative importance of the two
-  components via a adjustable paramter (see below)
+  components via a adjustable parameter (see below)
```
Comment (greglandrum) [inline Code/GraphMol/Depictor/RDDepictor.cpp]:
```suggestion
  components via an adjustable parameter (see below)
```

File: Code/ML/InfoTheory/InfoBitRanker.h [hunk 7]
```diff
@@ -209,7 +209,7 @@ class RDKIT_INFOTHEORY_EXPORT InfoBitRanker {
    *computed
    *    - the maximum of these fractions for classes that are not in the
    *biasList are computed
-   *    - If this maximum is less than the fraction for atleast one of classes
+   *    - If this maximum is less than the fraction for at least one of classes
```
Comment (greglandrum) [inline Code/ML/InfoTheory/InfoBitRanker.h]:
```suggestion
   *    - If this maximum is less than the fraction for at least one of the classes
```

File: Docs/Book/RDKit_Book.rst [hunk 8]
```diff
@@ -1348,7 +1348,7 @@ RDKit Fingerprints
 This is an RDKit-specific fingerprint that is inspired by (though it differs
 significantly from) public descriptions of the Daylight fingerprint
 [#daylightFP]_. The fingerprinting algorithm identifies all subgraphs in the
-molecule within a particular range of sizes, hashes each subraphs to generate a
+molecule within a particular range of sizes, hashes each subgraphs to generate a
```
Comment (greglandrum) [inline Docs/Book/RDKit_Book.rst]:
```suggestion
molecule within a particular range of sizes, hashes each subgraph to generate a
```

**Comment by greglandrum** [comment]
@e-kwsm : thanks for doing this!
It looks like you must have run a spelling/grammar checker across the code; how did you do that?

**Comment by e-kwsm** [comment]
> It looks like you must have run a spelling/grammar checker across the code; which tool did you use for that?

I used aspell and [typochecker](https://github.com/bwignall/typochecker).

**Pull request merged**